datadriven/src/sgpp/datadriven/application/learnersgdeonoffparallel/MPIMethods.cpp

```diff
@@ -758,8 +758,8 @@ void MPIMethods::processIncomingMPICommands(PendingMPIRequest &pendingMPIRequest
       break;
     case WORKER_SHUTDOWN_SUCCESS:std::cout << "Worker has acknowledged shutdown" << std::endl;
       break;
-    case NULL_COMMAND:std::cout << "Error: Incoming command has undefined command id" << std::endl;
-      throw sgpp::base::algorithm_exception("MPI_Packet with NULL command received.");
+    case nullptr_COMMAND:std::cout << "Error: Incoming command has undefined command id" << std::endl;
```

Comment:
I guess the MPI enums were not meant to be renamed

Reply:
That was one of two types of things I did via automation. Never trust a machine if you can do it yourself ;)